packer/build_ova.sh

```diff
@@ -0,0 +1,343 @@
+                                            ANSIBLE_PLAYBOOK=$1
+                                            ;;
+            --RACKHD_VERSION )              shift
+                                            RACKHD_VERSION=$1
+                                            ;;
+            --DEBIAN_REPOSITORY )           shift
+                                            DEBIAN_REPOSITORY=$1
+                                            ;;
+            --TARGET_DIR )                  shift
+                                            TARGET_DIR=$1
+                                            ;;
+            --CI_SIGNING_KEY )              shift
+                                            CI_SIGNING_KEY=$1
+                                            ;;
+            --GPG_SIGNING_KEY )             shift
+                                            GPG_SIGNING_KEY=$1
+                                            ;;
+            --CUSTOMIZED_PROPERTY_OVA )     shift
+                                            CUSTOMIZED_PROPERTY_OVA=$1
+                                            ;;
+            * )                             Usage
+                                            exit 1
+        esac
+        shift
+    done
+
+    if [ ! -n "${RACKHD_DIR}" ] ; then
+        echo "[Error]Arguments RACKHD_DIR is required!"
+        Usage
+        exit 1
+    fi
+
+    ####  Ubuntu Version  ###
+    if  [ ! -n "${OS_VER}" ];  then
+        OS_VER=ubuntu-14.04
+        echo "${INFO_HEADER} Packer OVA Build based on $OS_VER ( using template-${OS_VER}.json)"
+    fi
+
+    # By default. use ansible playbook : rackhd_local.yml
+    if  [ ! -n "${ANSIBLE_PLAYBOOK}" ];  then
+        ANSIBLE_PLAYBOOK=rackhd_local
+    fi
+    if [ ! -f "$RACKHD_DIR/packer/ansible/${ANSIBLE_PLAYBOOK}.yml" ]; then
+        echo "${ERROR_HEADER} The target ansible playbook($RACKHD_DIR/packer/ansible/${ANSIBLE_PLAYBOOK}.yml) does not exist. Aborting..."
+        exit 3
+    fi
+
+    ### $RACKHD_VERSION is used for package install provisioner:  `apt-get install rackhd=$RACKHD_VERSION` ###
+    if [  -n "${RACKHD_VERSION}" ];  then
+       if [ $ANSIBLE_PLAYBOOK != "rackhd_local" ]; then
+           echo "${INFO_HEADER} Build VM based on RackHD Debian Package ${RACKHD_VERSION} "
+      else
+           echo "${WARNING_HEADER} Skip the ENV VAR: RACKHD_VERSION=$RACKHD_VERSION, this variable is useless for rackhd_local.yml (build from src code)".
+       fi
+    else
+       if [ $ANSIBLE_PLAYBOOK != "rackhd_local" ]; then
+           echo "${INFO_HEADER} Install RackHD Package without specific version(default latest)."
+       fi
+    fi
+
+    if [  -n "${DEBIAN_REPOSITORY}" ];  then
+        if [ $ANSIBLE_PLAYBOOK != "rackhd_local" ]; then
+            echo "${WARNING_HEADER} Skip the ENV VAR: DEBIAN_REPOSITORY=$DEBIAN_REPOSITORY."
+        else
+            echo "${WARNING_HEADER} DEBIAN_REPOSITORY variable is useless for rackhd_local.yml (build from src code)".
```

Comment:
should be {INFO_HEADER} and missing a $ before DEBIAN_REPOSITORY

Reply:
1. I think this should be warning. User use rackhd local and use deb repo too. Maybe the user use the wrong yml file. So warning can help user to check this.
2. DEBIAN_REPOSITORY is var name not var value here.